src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+
+import java.util.Comparator;
+
+public class SortCommand extends Command {
```

Comment:
Coding standard requires to include descriptive header comments for public classes